### PR TITLE
Max Fee Calculation for Deposit, Transfer and Withdraw flows

### PR DESCRIPTION
**Pull request by AtelyPham**
## Summary of changes
- Removed deprecated fees method on relayer class
- Allow an address to fetch the currency balance
- Allow dest chain when calculating the balance from notes
- UI fixes

### Proposed area of change

- [x] `apps/bridge-dapp`
- [ ] `apps/stats-dapp`
- [ ] `apps/webbsite`
- [ ] `apps/faucet`
- [ ] `apps/tangle-website`
- [x] `libs/webb-ui-components`

### Reference issue to close (if applicable)
- Closes `NaN`

### Screen Recording

https://user-images.githubusercontent.com/60747384/230183883-58c76e2b-1a85-47fb-8e5c-225bad2847f9.mp4

### CODE REVIEW

File: libs/webb-ui-components/src/containers/DepositCard/DepositCard.tsx [hunk 1]
```diff
@@ -92,9 +95,8 @@ export const DepositCard = forwardRef<HTMLDivElement, DepositCardProps>(
 
             <InfoItem
               leftTextProps={{
-                title: `Fees ${feePercentage ? `(${feePercentage}%)` : ''}`,
+                title: feePercentage ? `Fees (${feePercentage}%)` : `Max fees`,
```
Comment (dutterbutter) [inline libs/webb-ui-components/src/containers/DepositCard/DepositCard.tsx:98]:
```suggestion
                title: feePercentage ? `Fee (${feePercentage}%)` : `Max fee`,
```

File: libs/webb-ui-components/src/containers/ConfirmationCard/WithdrawConfirm.tsx [hunk 2]
```diff
@@ -303,7 +303,7 @@ export const WithdrawConfirm = forwardRef<
               <InfoItem
                 leftTextProps={{
                   variant: 'body1',
-                  title: 'Estimated fees',
+                  title: 'Max fees',
```
Comment (dutterbutter) [inline libs/webb-ui-components/src/containers/ConfirmationCard/WithdrawConfirm.tsx:306]:
```suggestion
                  title: 'Max fee',
```

File: libs/webb-ui-components/src/containers/ConfirmationCard/TransferConfirm.tsx [hunk 3]
```diff
@@ -275,10 +284,9 @@ export const TransferConfirm = forwardRef<HTMLDivElement, TransferConfirmProps>(
               <InfoItem
                 leftTextProps={{
                   variant: 'body1',
-                  title: 'Fees',
-                  info: 'Fees',
+                  title: 'Max fees',
```
Comment (dutterbutter) [inline libs/webb-ui-components/src/containers/ConfirmationCard/TransferConfirm.tsx:287]:
```suggestion
                  title: 'Max fee',
```

**Review by dutterbutter — CHANGES_REQUESTED** [review]

File: apps/bridge-dapp/src/containers/TransferContainer/TransferContainer.tsx [hunk 4]
```diff
@@ -442,13 +481,28 @@ export const TransferContainer = forwardRef<
...
         return [];
       }
 
+      const typedChainId = calculateTypedChainId(
+        destChain.chainType,
+        destChain.chainId
+      );
+      const vanchorAddr = getLatestAnchorAddress(typedChainId);
+      if (!vanchorAddr) {
+        console.error('No anchor address found for chain', typedChainId);
```
[the remark this answers is not in the document]
Comment (AtelyPham) [reply apps/bridge-dapp/src/containers/TransferContainer/TransferContainer.tsx:494]: These errors will be on the browser console, but these are the development errors.

**Comment by devpavan04** [comment]
@AtelyPham I think we need to show type of token here.

Screenshot 2023-04-10 at 10 35 48 AM

**Pull request merged**